Source/Core/Core/IOS/ES/ES.cpp

```diff
@@ -303,24 +301,27 @@ IPCReply ESDevice::SetUID(u32 uid, const IOCtlVRequest& request)
 
   const u64 title_id = memory.Read_U64(request.in_vectors[0].address);
 
-  const s32 ret = CheckIsAllowedToSetUID(GetEmulationKernel(), uid, m_core.m_title_context.tmd);
+  auto& kernel = GetEmulationKernel();
+  ES::UIDSys uid_sys{kernel.GetFSCore()};
+  const s32 ret = CheckIsAllowedToSetUID(kernel, &uid_sys, uid, m_core.m_title_context.tmd);
   if (ret < 0)
   {
     ERROR_LOG_FMT(IOS_ES, "SetUID: Permission check failed with error {}", ret);
     return IPCReply(ret);
   }
 
-  const auto tmd = m_core.FindInstalledTMD(title_id);
+  u64 find_tmd_ticks = 0;
```

Comment:
Given the constructor says 'Please avoid using this function if more accurate timings are known.', it's confusing to me how 0 is more accurate in terms of timing.  It's not clear why you are doing this, maybe add a comment here?

Reply:
Can we also call this `delay_tmd_ticks`?

Reply:
how about just `delay_ticks` ?

Reply:
That works, thanks!~